### PR TITLE
[Japanese] Update Japanese specification to follow up to 2.0.0

### PR DESCRIPTION
**Pull request by ohbarye**
Compared to the original English document, I found that the Japanese one lacks some parts and there are some differences.

This commit makes Japanese specification follow up to https://github.com/semver/semver.org/commit/b5e2e75374611657c0c62e0bcc8d03172d49e3d5.

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
@ohbarye Thanks for contribution. Do you know any native speaker who can review these changes?

**Comment by ohbarye** [comment]
> Do you know any native speaker who can review these changes?

@munierujp @haacked @soukouki @gulping @foomin10 
How about you? I saw you've reviewed or contributed to this project in Japanese.

File: lang/ja/index.md [hunk 1]
```diff
@@ -106,17 +171,39 @@ FAQ
...
 
 　いいえ、ありませんが良識ある判断をしてください。例えば255文字のバージョン文字列は過剰と言えるでしょうし、特定のシステムではそれ独自の文字列の限界値があることでしょう。
 
+### 『v1.2.3』はセマンティック バージョンでしょうか？
+
+　いいえ、『v1.2.3』はセマンティック バージョンではありません。しかしながら、セマンティック バージョンに接頭辞の『v』を付けるのは英語ではバージョン番号であることを示す一般的な方法です。バージョン管理では、『バージョン』を『v』と略すことがよくあります。たとえば `git tag v1.2.3 -m" Release version 1.2.3 "` では『v1.2.3』はタグ名であり、セマンティック バージョンは『1.2.3』です。
+
+### SemVer文字列をチェックするために推奨される正規表現（RegEx）はありますか？
+
+　二つあります。一つは名前付きグループをサポートするシステム（PCRE [Perl Compatible Regular Expressions, i.e. Perl, PHP and R]、Python、Go）のための正規表現です。
```
Comment (munierujp) [inline lang/ja/index.md]: How about translating `i.e.` to `例：` or `〜などの` ?

File: lang/ja/index.md [hunk 2]
```diff
@@ -106,17 +171,39 @@ FAQ
...
+
+参考: <https://regex101.com/r/Ly7O1x/3/>
+
+```
+^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
+```
+
+　もう一つは番号付きキャプチャグループを使った正規表現です（したがって、cg1 = メジャー、cg2 = マイナー、cg3 = パッチ、cg4 = プレリリース、cg5 = ビルドメタデータを意味します）。これはECMA Script（JavaScript）、PCRE（Perl Compatible Regular Expressions, i.e. Perl, PHP and R）、Python、Goにおいて互換性があります。
```
Comment (munierujp) [inline lang/ja/index.md]: How about translating `i.e.` to `例：` or `〜などの` ?

**Review by munierujp — COMMENTED** [review]
Please read my comments.

File: lang/ja/index.md (hunk 1 above)
Comment (ohbarye) [reply lang/ja/index.md]: Sounds nice. Addressed in df34c714e6f82c8225753d884ab1a31c0759cc92.

File: lang/ja/index.md (hunk 2 above)
Comment (ohbarye) [reply lang/ja/index.md]: The same goes with https://github.com/semver/semver.org/pull/243#discussion_r439216499

**Comment by ohbarye** [comment]
@munierujp Thanks for your review, I've addressed your points!

File: lang/ja/index.md (hunk 1 above)
Comment (munierujp) [reply lang/ja/index.md]: @ohbarye Nice job! Could you update `lang/ja/spec/v2.0.0.md` too?

File: lang/ja/index.md (hunk 2 above)
Comment (munierujp) [reply lang/ja/index.md]: @ohbarye Nice job! Could you update `lang/ja/spec/v2.0.0.md` too?

File: lang/ja/index.md (hunk 1 above)
Comment (ohbarye) [reply lang/ja/index.md]: Oops, I updated both and rebased into 5cfd798. Thanks.

**Review by munierujp — APPROVED** [review]
It seems good for Japanese.

**Comment by alexandrtovmach** [comment]
@munierujp thanks

**Comment by ohbarye** [comment]
Thank both of you!

**Pull request merged**